crates/config/src/lib.rs

```diff
@@ -4442,4 +4446,35 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn test_parse_labels() {
+        figment::Jail::expect_with(|jail| {
+            jail.create_file(
+                "foundry.toml",
+                r#"
+                [labels]
```

Comment:
shouldn't the labels in config be chain specific though?

Reply:
I thought that most address labels for evm chains can be safely reused because contract addresses created using `CREATE2` and externally owned accounts created from same private key or seed phrase share the same address over all chains. Tagging `Uniswap V3: Factory` over all chains would be cumbersome. I think it would be much better if there is chain-specific label section and globally shared label section to keep chain-specific labels from universally used addresses, but I just kept this PR's scope into global labels only.